### ISSUE TITLE
Rename imported title

### ISSUE THREAD

**Issue by tjsuominen** [Pool-Of-Tears/Myne#191]
### Checklist

- [X] This issue contains only one feature request. I will open one issue for every feature I want to request.
- [X] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo.
- [ ] This feature is merely an UI/UX update.
- [X] This feature is not going to conflict with many of the existing features/options.

### Is your feature request related to a problem? Please describe.

No

### Describe the solution you'd like

Ability to rename an imported ePub

### How will you/everyone benefit from this feature?

Sometimes imported ePubs have strange / mixed name. Ability to rename the pub is useful.

**Comment by starry-shivam** [comment]
> Sometimes imported ePubs have strange / mixed name. Ability to rename the pub is useful.

What do you mean by "strange/mixed name"? Does it cause any trouble when importing? If yes, then this should be filed as a bug report.

The book's name is taken from the `dc:title` attribute of the book's metadata (manifest) file. If you want a different name, you can edit the manifest by extracting the EPUB archive. Manifest files are usually located at `OEBPS/content.opf`.

**Comment by tjsuominen** [comment]
Yes, the title was read correctly (by Myne), but would like to have the ability to rename the Titles after importing rather than extracting manually, edit, re-import.

**Comment by starry-shivam** [comment]
> Yes, the title was read correctly (by Myne), but would like to have the ability to rename the Titles in Myne after importing; rather than extracting manually, edit, re-import.

I see, I'll think about it.

**Comment by starry-shivam** [comment]
I'm closing this as i don't think it makes much sense after 2nd thought
